src/tools/react-native.ts

```diff
@@ -176,3 +176,41 @@ export async function renameReactNativeApp(
     }),
   )
 }
+
+export async function replaceMaestroBundleIds(
+  toolbox: GluegunToolbox,
+  oldBundleIdentifier: string,
+  newBundleIdentifier: string,
+) {
+  const { parameters, filesystem, print } = toolbox
+  const { path } = filesystem
+
+  // debug?
+  const debug = Boolean(parameters.options.debug)
+  const log = <T = unknown>(m: T): T => {
+    debug && print.info(` ${m}`)
+    return m
+  }
+
+  // here's a list of all the files to patch the name in
+  const filesToPatch = [`.maestro/Login.yaml`, `.maestro/FavoritePodcast.yaml`]
```

Comment:
Not mission critical, but I think using a glob here would be good because I'm definitely not going to remember to update this if we add any tests in future PRs.

```suggestion
  const filesToPatch = filesystem
      .cwd(TARGET_DIR)
      .find({
        matching: `.maestro/**.yaml`,
        files: true,
        directories: false,
      })
```